docs/serial_recovery.md

```diff
@@ -0,0 +1,39 @@
+<!--
+  - SPDX-License-Identifier: Apache-2.0
+
+  - Copyright (c) 2022 Nordic Semiconductor ASA
+
+  - Original license:
+
+  - Licensed to the Apache Software Foundation (ASF) under one
+  - or more contributor license agreements.  See the NOTICE file
+  - distributed with this work for additional information
+  - regarding copyright ownership.  The ASF licenses this file
+  - to you under the Apache License, Version 2.0 (the
+  - "License"); you may not use this file except in compliance
+  - with the License.  You may obtain a copy of the License at
+
+  -  http://www.apache.org/licenses/LICENSE-2.0
+
+  - Unless required by applicable law or agreed to in writing,
+  - software distributed under the License is distributed on an
+  - "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  - KIND, either express or implied.  See the License for the
+  - specific language governing permissions and limitations
+  - under the License.
+-->
+
+# Serial Recovery
+
+Mcuboot implements a Simple Management Protocol (SMP) server. SMP is a basic transfer encoding for use with the MCUmgr management protocol.
+For more information about MCUmgr and SMP. Please refer [the zephyr-rtos device managemnt documenation](https://docs.zephyrproject.org/latest/services/device_mgmt/index.html#device-mgmt) for detail of these components.
+
+Limited subset of MCUmgr comands are suported:
+- image list
+- image upload
+...
+
+Serial recovery feature might use any serial interface provided by the platform.
```

Comment:
It would be useful to point out that the DTS needs to have a `zephyr,uart-mcumgr` entry.

Reply:
> It would be useful to point out that the DTS needs to have a zephyr,uart-mcumgr entry.

Well, for Zephyr, that is!